src/components/ResourceBrowser/ResourceList/ResourceBrowserActionMenu.tsx

```diff
@@ -65,6 +65,7 @@ const ResourceBrowserActionMenu: React.FC<ResourceBrowserActionMenuType> = ({
     getResourceListData,
     handleResourceClick,
     removeTabByIdentifier,
+    hideDeleteResource,
```

Comment:
Should we enable it as we have the GVK?

Reply:
Not right now, since the API takes too much time and we need to refresh the data on delete